frontend/src/components/RotateCredentials.vue

```diff
@@ -0,0 +1,434 @@
+        return true
+      }
+      if (this.isScheduled) {
+        return true
+      }
+      if (this.isProgressing && this.type !== 'ALL_CREDENTIALS') {
+        // Only show the loading indicator for the rotation that is actually running, not for the overall trigger button
+        return true
+      }
+      return false
+    },
+    isDisabled () {
+      if (this.phase && this.phase.incomplete) {
+        return true
+      }
+      if (this.isHibernationPreventingRotation) {
+        return true
+      }
+      if (this.isScheduledOperation) {
+        return true
+      }
+      if (this.isProgressing) {
+        return true
+      }
+      return false
+    },
+    isScheduledForMaintenance () {
+      return this.shootAnnotations['maintenance.gardener.cloud/operation'] === this.operation
+    },
+    isScheduled () {
+      return this.shootAnnotations['gardener.cloud/operation'] === this.operation
+    },
+    isScheduledOperation () {
+      return !!this.shootAnnotations['gardener.cloud/operation']
+    },
+    isMaintenanceDisabled () {
+      return !this.isScheduledForMaintenance && !!this.shootAnnotations['maintenance.gardener.cloud/operation']
+    },
+    maintenanceHint () {
+      if (this.isMaintenanceDisabled) {
+        return 'Another operation has already been scheduled. Only one operation at a time can be scheduled.'
+      }
+      return ''
+    },
+    isHibernationPreventingRotation () {
+      return this.isShootStatusHibernated &&
+        includes(['rotate-credentials-start',
+          'rotate-etcd-encryption-key-start',
+          'rotate-credentials-complete',
+          'rotate-etcd-encryption-key-complete',
+          'rotate-serviceaccount-key-start',
+          'rotate-serviceaccount-key-complete'],
+        this.operation)
+    },
+    icon () {
+      if (this.isScheduledForMaintenance) {
+        return 'mdi-clock-outline'
+      }
+      return 'mdi-refresh'
+    },
+    confirmButtonText () {
+      if (this.isScheduledForMaintenance && !this.maintenance) {
+        return 'Unschedule Rotation'
+      }
+      if (this.maintenance) {
+        return 'Schedule Roatation'
+      }
+      switch (this.mode) {
+        case 'START':
+          return 'Prepare Roatation'
+        case 'COMPLETE':
+          return 'Complete Rotation'
+      }
+      return 'Rotate'
+    },
+    tooltip () {
+      if (this.isScheduledForMaintenance) {
+        return 'This operation is scheduled to be performed during the next maintenance time window'
+      }
+      if (this.isHibernationPreventingRotation && this.isShootStatusHibernated) {
+        return 'Cluster is hibernated. Wake up cluster to perform this operation'
+      }
+      if (this.phase && this.phase.incomplete) {
+        return 'Operation is disabled because all two-step operations need to be in the same phase'
+      }
+      if (this.isScheduledOperation) {
+        return 'There is alread an operation scheduled for this cluster'
```

Comment:
```suggestion
        return 'There is already an operation scheduled for this cluster'
```

Reply:
<img width="1160" alt="Screenshot 2022-11-02 at 16 59 00" src="https://user-images.githubusercontent.com/5526658/199539414-dd5f9554-68c6-47e9-a352-36e43b87fbe9.png">
this tooltip on the spinner icon does not make sense. Instead it should say that the operation is in progress / scheduled

Reply:
done